application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -52,6 +62,10 @@ public final class ReminderCommand extends SlashCommandAdapter {
     private static final List<String> TIME_UNITS =
             List.of("minutes", "hours", "days", "weeks", "months", "years");
     private static final Period MAX_TIME_PERIOD = Period.ofYears(3);
+    private static final int REMINDERS_PER_PAGE = 10;
+    private static final int MAX_REMINDER_TITLE_LENGTH = 256;
+    private static final Emoji PREVIOUS_BUTTON_EMOJI = Emoji.fromUnicode("⬅");
+    private static final Emoji NEXT_BUTTON_EMOJI = Emoji.fromUnicode("➡");
```

Comment:
but this is emoji y call it label

Reply:
so that people understand what its used for. its clear that its an emoji, the type already says that. thats boring. the important bit is "okay, what is this emoji used for?" its used as a label for the button.

Reply:
I agree with Taz's naming. Buttons can have a label **and** an emoji. Calling this label would be wrong

Reply:
fair